### ISSUE TITLE
Test on my device

### ISSUE THREAD

**Issue by Nepenthes-zlc** [SeungjunNah/DeepDeblur-PyTorch#47]
Hi @SeungjunNah !
I'm so appreciated you provide us with your work to learn more.I want to test the work on my device .As I know , the current result shows that everything loads from the downloaded files,including the loss ,psnr and ssim.What I want to do is that after loading the model weight that you provide, then test on my device and result new loss, psnr and ssim.I want to know is it can be done? If so ,could you help me?

**Comment by SeungjunNah** [comment]
Hi @keys-zlc 
You can start training with a new `--save_dir` argument and load weight by specifying the location with `--pretrained` argument.  
https://github.com/SeungjunNah/DeepDeblur-PyTorch/blob/master/src/option.py#L52

For example, 
```bash
python main.py --save_dir NEW_DIRNAME --pretrained MODEL_PATH.pt
```

**Comment by Nepenthes-zlc** [comment]
Thanks for your reply. Sorry, maybe I haven't expressed my question clearly. What I want to do is not to start training, but to test. In fact, I want to load the weight you provided which names model-1000.pt and then test on the GOPRO to get the PSNR and SSIM. I divide my question into pieces:

1.  I have downloaded the weight you provided. If I run the command 
"python main.py --save_dir GOPRO_L1 --dataset GOPRO_Large --do_train false --do_test true --test_every 1 --load_epoch 1000 --start_epoch 1000"
I find it seems just load the existed PSNR and SSIM, and doesn't run again on the GOPRO test. I'm not sure whether it is true. 
2. If it doesn't run again on the GOPRO test exactly, I want to just run it again using your weight model-1000.pt without training again.
I don't known if I express it clearly. Looking forward to your reply!

**Comment by SeungjunNah** [comment]
I tried the following command and it produces the test images and measures new PSNR and SSIM.
```bash
python main.py --save_dir GOPRO_L1 --dataset GOPRO_Large --do_train false --do_test true --load_epoch 1000 --start_epoch 1000 --end_epoch 1000 --save_results all 
```

You will see the numbers changing in the progress bar. They are average PSNR, SSIM from 0 to the processed image.

**Comment by Nepenthes-zlc** [comment]
Thanks, it works well!
